pom.xml

```diff
@@ -108,7 +108,7 @@
         <muleWsdl4j.version>1.6.4</muleWsdl4j.version>
         <slf4j.version>1.7.29</slf4j.version>
         <apache.commons.lang3.version>3.10</apache.commons.lang3.version>
-        <cxfRtWsdlVersion>3.3.6</cxfRtWsdlVersion>
+        <cxfRtWsdlVersion>2.7.19-MULE-006</cxfRtWsdlVersion>
```

Comment:
Reverting back to CXF 2 which is out of support? Why?

Reply:
Do we have one within support?